docs/node/run-your-node/paratime-node.mdx

```diff
@@ -264,66 +264,59 @@ If a ParaTime requires the use of a TEE, then make sure you set up TEE as instru
 In order to configure the node create the `/node/etc/config.yml` file with the following content:
 
 ```yaml
-datadir: /node/data
-
-log:
-  level:
-    default: info
-    tendermint: info
-    tendermint/context: error
-  format: JSON
-
-genesis:
-  file: /node/etc/genesis.json
+mode: compute
+common:
+    data_dir: /node/data
+    log:
+        format: JSON
+        level:
+            cometbft: info
+            cometbft/context: error
+            default: info
 
 consensus:
-  tendermint:
-    core:
-      listen_address: tcp://0.0.0.0:26656
+    # The external IP that is used when registering this node to the network.
+    # NOTE: If you are using the Sentry node setup, this option should be
+    # omitted.
+    external_address: tcp://{{ external_address }}:26656
+    listen_address: tcp://0.0.0.0:26656
 
-      # The external IP that is used when registering this node to the network.
-      # NOTE: If you are using the Sentry node setup, this option should be
-      # omitted.
-      external_address: tcp://{{ external_address }}:26656
+genesis:
+    file: /node/etc/genesis.json
 
-    p2p:
-      # List of seed nodes to connect to.
-      # NOTE: You can add additional seed nodes to this list if you want.
-      seed:
-        - "{{ seed_node_address }}"
+# The following section is required for ParaTimes which are running inside the
```

Comment:
```suggestion
# The IAS section is required for ParaTimes which are running inside the
```

Or move the IAS section at the end, like it was before. Since currently it's not clear that "The following section" is ment only fort the "IAS" part.